Tests/test_attribute_customize.py

```diff
@@ -28,245 +29,249 @@ def return_100(self, name):        return 100
+        self.assertEqual(c.x, 30)
 
-    c = C()
-    C.__getattr__ = lambda *args: 20
-    
-    C.__getattribute__ = lambda *args: 30
-    AreEqual(c.x, 30)
+        C.__getattribute__ = throw_attribute_error
+        self.assertEqual(c.x, 20)
 
-    C.__getattribute__ = throw_attribute_error
-    AreEqual(c.x, 20)
+        C.__getattribute__ = throw_assertion_error
+        self.assertRaises(AssertionError, lambda: c.x)
 
-    C.__getattribute__ = throw_assertion_error
-    AssertError(AssertionError, lambda: c.x)
+        C.__getattribute__ = lambda *args: C.__getattr__(*args)
+        self.assertEqual(c.x, 20)
 
-    C.__getattribute__ = lambda *args: C.__getattr__(*args)
-    AreEqual(c.x, 20)
+    def test_subclassing(self):
+        C1, C2 = new_classes()
+        
+        ## new style
+        class D(C2): pass
+        d = D()
+        d.x = 10
+        
+        C2.__getattr__ = return_100
+        self.assertEqual(d.y, 100)
+        del C2.__getattr__
+        
+        def f(self, name, value): self.__dict__[name] = value + 10
+        C2.__setattr__ = f
+        d.x = 20
+        self.assertEqual(d.x, 30)
+        del C2.__setattr__
+        
+        C2.__getattribute__ = return_100
+        #self.assertEqual(d.x, 100)  # bug 365242
+        
+        ## old style
+        class D(C1): pass
+        d = D()
+        C1.__getattr__ = return_100
+        #self.assertRaises(AttributeError, lambda: d.y)  # (no?) dynamism for old style, bug 365266
+        
+        class D(C1): pass
+        d = D()
+        d.x = 10
+        self.assertEqual([d.x, d.y], [10, 100])
+        
+        C1.__setattr__ = f
+        class D(C1): pass
+        d = D()
+        d.x = 20
+        self.assertEqual([d.x, d.y], [30, 100])
+        
+        C1.__getattribute__ = lambda *args: 200  # __getattribute__ not honored
+        class D(C1): pass
+        d = D()
+        self.assertEqual([d.x, d.y], [100, 100])
 
-def test_subclassing():
-    C1, C2 = new_classes()
-    
-    ## new style
-    class D(C2): pass
-    d = D()
-    d.x = 10
-    
-    C2.__getattr__ = return_100
-    AreEqual(d.y, 100)
-    del C2.__getattr__
-    
-    def f(self, name, value): self.__dict__[name] = value + 10
-    C2.__setattr__ = f
-    d.x = 20
-    AreEqual(d.x, 30)
-    del C2.__setattr__
-    
-    C2.__getattribute__ = return_100
-    #AreEqual(d.x, 100)  # bug 365242
-    
-    ## old style
-    class D(C1): pass
-    d = D()
-    C1.__getattr__ = return_100
-    #AssertError(AttributeError, lambda: d.y)  # (no?) dynamism for old style, bug 365266
-    
-    class D(C1): pass
-    d = D()
-    d.x = 10
-    AreEqual([d.x, d.y], [10, 100])
-    
-    C1.__setattr__ = f
-    class D(C1): pass
-    d = D()
-    d.x = 20
-    AreEqual([d.x, d.y], [30, 100])
-    
-    C1.__getattribute__ = lambda *args: 200  # __getattribute__ not honored
-    class D(C1): pass
-    d = D()
-    AreEqual([d.x, d.y], [100, 100])
+    def test_delete_getattribute(self):
```

Comment:
Yes, it runs correctly. I generally have been enabling all the tests and then looking through to see if there really is something that should be disabled. I am also trying to use expectedFailure instead of disabling if there is an issue, that way it is noted in the logs.